### ISSUE TITLE
ELAN Touchpad not working on Pop 20.04

### ISSUE THREAD

**Issue by DrRSatzteil** [pop-os/pop#1071]
**Distribution (run `cat /etc/os-release`):**
NAME="Pop!_OS"
VERSION="20.04 LTS"

**Issue/Bug Description:**
Same problem as described in #724. The touchpad of my Ideapad 5 does not work at all. However the proposed fix in #724 does not work.

**Steps to reproduce (if you know):**
Plain install of Pop 20.04

**Expected behavior:**
Touchpad is working

**Other Notes:**

**Comment by DrRSatzteil** [comment]
Found this interesting thread in the lenovo support forums:
https://forums.lenovo.com/topic/view/2713/5017099

However I wasn't able to compile my own kernel so far to try whether the suggestion in the thread works...

**Comment by DrRSatzteil** [comment]
Update: managed to compile my own kernel with Elan drivers set to module build. It did not work (did a couple of reboots). Then I booted up a Fedora Live System where the touchpad was working. After another reboot the touchpad is also working in Pop OS. Tried to go back to the old kernel: Touchpad does not work. Back to my new kernel: Works! So the fix suggested in the lenovo forums seems to work even though I cannot explain why it did not work until I booted up another live linux...
